docs/source/getting-started/content/proc_basic-troubleshooting.adoc

```diff
@@ -1,6 +1,20 @@
 [id="basic-troubleshooting_{context}"]
 = Basic troubleshooting
 
+[NOTE]
+====
+Each released `crc` binary expires 30 days after the release due to certificates embedded in the Openshift cluster.
+Using an older `crc` binary will result in an expired certificates error.
+
+To resolve this error:
+
+* link:https://github.com/code-ready/crc/releases[Download the latest CodeReady Containers release] and place the `crc` binary in your `$PATH`.
+* Remove the cluster with certificate errors using the `crc delete` command.
+* Set up the new release with the `crc setup` command.
+* Start the new cluster with the `crc start` command.
+
+====
+
 The majority of issues can be resolved by stopping a running {prod} virtual machine, deleting the virtual machine, and starting a new instance of the virtual machine.
 
```

Comment:
These are the same steps as the ones you detailed just above, we could turn what you added in generic instructions that would go there, and then we would refer to these steps  inthe "certificate recovery" description?